newdle/client/src/reducers/creation.js

```diff
@@ -66,6 +75,12 @@ export default combineReducers({
   },
   participants: (state = [], action) => {
     switch (action.type) {
+      case NEWDLE_RECEIVED:
+        // TODO: remove condition when initials get settled
+        // This condition exists because a no-permission returns null
+        return action.newdle.participants
+          ? action.newdle.participants.map(p => ({...p, initials: 'AB'}))
+          : [];
```

Comment:
RFC: Can we get rid of this?
User search results (when creating a newdle) return a user object that contains an `initials` attribute derived from the first and last name. Participants do not. What should we do here? Add a dynamic field in the schema that does it for us? Or, even better, generate one in the client?

Eventually this would become just: `return action.newdle.participants`.

Reply:
Should we just ditch the two-char initials in favor of a simple `name[0]`? It's impossible to generate initials from a single name field since we don't know what's the first/last name.

Reply:
We could divide the space and try to figure out the initials, or fallback to the first two chars. How does this sound?

Reply:
single char is more reliable IMO - especially at CERN you have so many people with multiple first and last names...

Reply:
Only realized we were talking about a single char, after replacing it to be the first two chars of the name. It seems good, nonetheless.